### ISSUE TITLE
False positive: "Parsing error: Expected a pseudo-class or pseudo-element" when using Sass property nesting  

### ISSUE THREAD

**Issue by JoCa96** [future-architect/eslint-plugin-vue-scoped-css#371]
There is a false positive for the rule `vue-scoped-css/no-parsing-error`, when using SCSS and the [Property Declaration Nesting](https://sass-lang.com/documentation/style-rules/declarations/#nesting) feature.

```vue 
<style lang="scss"> // <-- Parsing error: Expected a pseudo-class or pseudo-element. eslint(vue-scoped-css/no-parsing-error)
h1 {
  border: {
    style: solid;
    width: var(--onyx-1px-in-rem);
    color: var(--onyx-radio-button-selector-border-color);
    radius: var(--onyx-radius-full);
  }
}
</style>

```
